index.html

```diff
@@ -30,144 +42,160 @@ <h2 class="connect">LET'S CONNECT</h2>
-            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
-                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
-                contact me.</p>
+    </div>
+</section>
+<section class="container">
+    <div>
+    <img  class="image" src="assets/img/snapshot-2.jpg">
+    <div class="float">
+    <ul class="">
+    <li class="tonic">Multi-Post Stories</li><br>
+    </ul>
+    <div>
+    <ul class="items">
+        <li class="canopy">CANOPY</li>
+        <li class="list">Full Stack Dev</li>
+        <li class="list1">2015</li>
+    </ul>
+    </div>
+    </div>
+    <p class="para">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+    <ul class="tools">
+    <li class="list2">html</li>
+    <li class="list2">Ruby on rails</li>
+    <li class="list2">css</li>
+    <li class="list2">javaScript</li>
+</ul>
+<div>
+    <button type="button" class="button-1">See Project</button>
+</div>
+</div>
+</section>
+    <section class="container">
+        <div>
+        <img src="assets/img/snapshoot-1.jpg">
+        <div class="float">
+        <ul class="">
+        <li class="tonic">Facebook 360</li><br>
+        </ul>
+        <div>
+        <ul class="items">
+            <li class="canopy">FACEBOOK</li>
+            <li class="list">Full Stack Dev</li>
+            <li class="list1">2015</li>
+        </ul>
         </div>
-        <h2 class="connect">LET'S CONNECT</h2>
-        <ul class="icons">
+        </div>
+        <p class="para">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+        <ul class="tools">
+        <li class="list2">html</li>
+        <li class="list2">Ruby on rails</li>
+        <li class="list2">css</li>
+        <li class="list2">javaScript</li>
+    </ul>
+    <div>
+        <button type="button" class="button-1">See Project</button>
+    </div>
+</div>
+</section>
+<section class="container">
+    <div>
+    <img  class="image" src="assets/img/snapshot-4.jpg">
+    <div class="float">
+    <ul class="">
+    <li class="tonic">Uber Navigation</li><br>
+    </ul>
+    <div>
+    <ul class="items">
+        <li class="canopy">Uber</li>
+        <li class="list">Lead Developer</li>
+        <li class="list1">2018</li>
+    </ul>
+    </div>
+    </div>
+    <p class="para">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+    <ul class="tools">
+    <li class="list2">html</li>
+    <li class="list2">Ruby on rails</li>
+    <li class="list2">css</li>
+    <li class="list2">javaScript</li>
+</ul>
+<div>
+    <button type="button" class="button-1">See Project</button>
+</div>
+</div>
+</section>
+    <section class="contact">
```

Comment:
Kindly set your` contact section` to cover the whole page in accordance to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design.
| Your Design |Figma Design |
| :---:   | :---: | 
|![image](https://user-images.githubusercontent.com/73854999/200121591-890a9aba-bab2-41ab-aa90-a66e3137352c.png)|![image](https://user-images.githubusercontent.com/73854999/200121557-a79e2c4f-770b-4d73-97e7-9f0a84141c05.png)|